### PR TITLE
Set up Mobile client 

### PR DESCRIPTION
**Pull request by Carltesio**
# [PT Story](https://www.pivotaltracker.com/story/show/172052286)
## Changes proposed in this pull request:
* Adds scaffold for ionic mobile app 
* Adds a basic rendering page of a menu.

## What I have learned working on this feature:
* Not to forget to create .gitignore file 
* Create a basic Ionic app

## Screenshots (Only if client side story)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import './theme/variables.css';
+
+const App = () => (
+  <IonApp>
+    
+      <IonHeader>
+      <Header/>
+      </IonHeader>
+     <IonContent>
+     <Menu />
```
Comment (emtalen) [inline src/App.jsx]: formatting

File: src/components/ExploreContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+const ExploreContainer: React.FC<ContainerProps> = () => {
+  return (
+    <div className="container">
+      <strong>Ready to create an app?</strong>
+      <p>Start with Ionic <a target="_blank" rel="noopener noreferrer" href="https://ionicframework.com/docs/components">UI Components</a></p>
+    </div>
+  );
+};
+
+export default ExploreContainer;
```
Comment (emtalen) [inline src/components/ExploreContainer.tsx]: You can probably remove this file

File: src/components/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  IonItem,
+  IonSegment
+} from '@ionic/react';
+
+
+
+const Header = () => {
+  return (
+    <IonSegment>
+  Menu Item
```
Comment (emtalen) [inline src/components/Header.jsx]: formatting

File: src/components/Articles.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+          <IonItem>
+            <IonIcon icon={basketball} slot="start" />
+            <IonLabel>Title snippet</IonLabel>
+            <IonButton fill="outline" slot="end">View</IonButton>
+          </IonItem>
+
+          <IonCardContent>
+        This is some snippet
+      </IonCardContent>
+        </IonCard>
```
Comment (emtalen) [inline src/components/Articles.jsx:20]: formatting

File: src/pages/Home.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import { IonContent, IonHeader, IonPage, IonTitle, IonToolbar } from '@ionic/react';
```
Comment (pgaunitz) [inline src/pages/Home.tsx]: rename file to .jsx

File: src/components/ExploreContainer.tsx [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```
Comment (pgaunitz) [inline src/components/ExploreContainer.tsx]: .jsx

**Review by pgaunitz — CHANGES_REQUESTED** [review]

**Review by pgaunitz — APPROVED** [review]

**Pull request merged**